crates/hyperqueue/src/client/task.rs

```diff
@@ -53,6 +55,19 @@ pub struct TaskInfoOpts {
     pub verbosity: VerbosityFlag,
 }
 
+#[derive(clap::Parser)]
+pub struct TaskExplainOpts {
+    /// Select specific job
+    #[arg(value_parser = parse_last_single_id)]
+    pub job_selector: SingleIdSelector,
+
+    /// Select specific task(s)
```

Comment:
Can you actually select multiple tasks with this?

Reply:
fixed